packages/stream-metadata/src/routes/spaceMetadata.ts

```diff
@@ -40,15 +45,7 @@ export async function fetchSpaceMetadata(request: FastifyRequest, reply: Fastify
 	}
 
 	const { spaceAddress } = parseResult.data
-
-	// Validate spaceAddress format using the helper function
-	if (!isValidEthereumAddress(spaceAddress)) {
-		logger.info({ spaceAddress }, 'Invalid spaceAddress format')
-		return reply
-			.code(400)
-			.header('Cache-Control', CACHE_CONTROL['4xx'])
-			.send({ error: 'Bad Request', message: 'Invalid spaceAddress format' })
-	}
+	logger.info({ spaceAddress }, 'Fetching space metadata')
```

Comment:
does this result in us logging every request twice?

Reply:
Technically yes. We log all incoming requests in a middleware, but since we also have this logging pattern in other endpoints, I would find it useful to have the same here